src/patternfly/components/Pagination/pagination.scss

```diff
@@ -112,6 +113,11 @@
       box-shadow: none;
     }
 
+    &.pf-m-sticky {
```

Comment:
Could we also make it top sticky in case someone wants to do that?

Also I realized I left `bottom: 0;` on the bottom pagination on the desktop view when the pagination isn't sticky. We should probably add `bottom: auto` there, then reset `bottom: 0` on `.pf-m-bottom.pf-m-sticky`.